build.win32x86/pharo.stack.spur/Makefile

```diff
@@ -0,0 +1,38 @@
+#############################################################################
+# Makefile for Win32 Stack Spur PharoVM using gcc and cygwin
+# Do make init to allow make -n to function.
+#############################################################################
+
+VM:=Pharo
+VM_NAME:=Pharo Virtual Machine
+
+VMSRCDIR:=../../spurstacksrc/vm
+# NOTES: 
+# STACK_ALIGN_BYTES=16 is needed in mingw and FFI (and I suppose on other modules too).
+# ALLOCA_LIES_SO_USE_GETSP=0 Some compilers return the stack address+4 on alloca function, 
+# then FFI module needs to adjust that. It is NOT the case of mingw.
+# For more information see this thread: http://forum.world.st/There-are-something-fishy-with-FFI-plugin-td4584226.html
+#COGDEFS:= -DPharoVM=1 -DSTACK_ALIGN_BYTES=16 -DALLOCA_LIES_SO_USE_GETSP=0

```

Comment:
These comments are troubling...
Either these flags are necessary as the comment tells... but then why don't we find them in code?
... or they are not anymore to date and should be removed.
I know you did not add these lines by yourself, only copied them, but someone forgot to clean up, so while at it...
(All these changes are marked as authored by Esteban, but I'm pretty sure that it was me who did perform the cleanup; at the time when wanting to work in a parallel pharo repository, git was somehow missused!)

Reply:
Or maybe I just gave instructions in commit message e9ff2f20501ee86801747f06f653fd669e138de9 ?
Failure to cleanup is not only mine, it's collective then ;)

Reply:
you are right, I just copied those lines from pharo.cog.spur.
But is this PR the place to fix these? This PR is focused on build.win64x64/pharo.cog.spur and your concern seems a broader issue...

```
$ find opensmaltalk-vm   -name Makefile  -print  -exec grep STACK_ALIGN_BYTES {} \;

./build.win32x86/pharo.cog.spur/Makefile
# STACK_ALIGN_BYTES=16 is needed in mingw and FFI (and I suppose on other modules too).
#COGDEFS:= -DPharoVM=1 -DSTACK_ALIGN_BYTES=16 -DALLOCA_LIES_SO_USE_GETSP=0

./build.win32x86/pharo.cog.spur.lowcode/Makefile
# STACK_ALIGN_BYTES=16 is needed in mingw and FFI (and I suppose on other modules too).
#COGDEFS:= -DPharoVM=1 -DSTACK_ALIGN_BYTES=16 -DALLOCA_LIES_SO_USE_GETSP=0

./build.win32x86/pharo.sista.spur/Makefile
# STACK_ALIGN_BYTES=16 is needed in mingw and FFI (and I suppose on other modules too).
#COGDEFS:= -DPharoVM=1 -DSTACK_ALIGN_BYTES=16 -DALLOCA_LIES_SO_USE_GETSP=0

./build.win32x86/squeak.cog.spur.lowcode/Makefile
COGDEFS:= -DSTACK_ALIGN_BYTES=16 -DALLOCA_LIES_SO_USE_GETSP=0
```

Reply:
Of course, changes can be post-poned after this PR. i just said "_while at _it__"